### PR TITLE
feat(auth): add content for token revocation when signing out

### PR DESCRIPTION
**Pull request by div5yesh**
_Issue #, if available:_

_Description of changes:_

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/lib/auth/fragments/native_common/signout/common.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ Invoke the `signOut` api to sign out a user from the Auth category. You can only
 
 Calling signOut without any options will just delete the local cache and keychain of the user. If you would like to sign out of all devices, invoke the signOut api with advanced options.
 
+[Amazon Cognito now supports token revocation](https://aws.amazon.com/about-aws/whats-new/2021/06/amazon-cognito-now-supports-targeted-sign-out-through-refresh-token-revocation/)  and latest Amplify version will revoke Amazon Cognito tokens if the application is online. This means Cognito refresh token cannot be used anymore to generate new Access and Id Tokens.
```
Comment (siegerts) [inline docs/lib/auth/fragments/native_common/signout/common.md:9]:
```suggestion
[Amazon Cognito now supports token revocation](https://aws.amazon.com/about-aws/whats-new/2021/06/amazon-cognito-now-supports-targeted-sign-out-through-refresh-token-revocation/) and the latest Amplify version will revoke Amazon Cognito tokens if the application is online. This means that the Cognito refresh token cannot be used anymore to generate new Access and Id Tokens.
```

File: docs/lib/auth/fragments/native_common/signout/common.md [hunk 2]
```diff
@@ -6,6 +6,16 @@ Invoke the `signOut` api to sign out a user from the Auth category. You can only
...
+
+Access and Id Tokens are short-lived (60 minutes by default but can be set from 5 minutes to 1 day). After revocation these tokens cannot be used with Cognito User Pools anymore, however they are still valid when used with other services like AppSync or API Gateway.
```
Comment (siegerts) [inline docs/lib/auth/fragments/native_common/signout/common.md:11]:
```suggestion
Access and Id Tokens are short-lived (60 minutes by default but can be set from 5 minutes to 1 day). After revocation, these tokens cannot be used with Cognito User Pools anymore. However, they are still valid when used with other services like AppSync or API Gateway.
```

File: src/fragments/lib/auth/native_common/signout/common.mdx [hunk 3]
```diff
@@ -14,6 +14,16 @@ import flutter2 from "/src/fragments/lib/auth/flutter/signout/10_local_signout.m
...
 
+[Amazon Cognito now supports token revocation](https://aws.amazon.com/about-aws/whats-new/2021/06/amazon-cognito-now-supports-targeted-sign-out-through-refresh-token-revocation/) and the latest Amplify version will revoke Amazon Cognito tokens if the application is online. This means that the Cognito refresh token cannot be used anymore to generate new Access and Id Tokens.
+
+Access and Id Tokens are short-lived (60 minutes by default but can be set from 5 minutes to 1 day). After revocation, these tokens cannot be used with Cognito User Pools anymore. However, they are still valid when used with other services like AppSync or API Gateway.
+
+For limiting subsequent calls to these other services after invalidating tokens, we recommend lowering token expiration time for your app client in the Cognito User Pools console. If you are using the Amplify CLI this can be accessed by running `amplify console auth`.
+
+Token revocation is enabled automatically on new Amazon Cognito User Pools, however existing User Pools must enable this feature, [using the Cognito Console or AWS CLI](https://docs.aws.amazon.com/cognito/latest/developerguide/token-revocation.html)
```
Comment (tjleing) [inline src/fragments/lib/auth/native_common/signout/common.mdx:23]: Maybe
```suggestion
Token revocation is enabled automatically on new Amazon Cognito User Pools, however existing User Pools must enable this feature, [using the Cognito Console or AWS CLI](https://docs.aws.amazon.com/cognito/latest/developerguide/token-revocation.html).
```

File: docs/lib/auth/fragments/native_common/signout/common.md [hunk 4]
```diff
@@ -6,6 +6,16 @@ Invoke the `signOut` api to sign out a user from the Auth category. You can only
...
 
+[Amazon Cognito now supports token revocation](https://aws.amazon.com/about-aws/whats-new/2021/06/amazon-cognito-now-supports-targeted-sign-out-through-refresh-token-revocation/) and the latest Amplify version will revoke Amazon Cognito tokens if the application is online. This means that the Cognito refresh token cannot be used anymore to generate new Access and Id Tokens.
+
+Access and Id Tokens are short-lived (60 minutes by default but can be set from 5 minutes to 1 day). After revocation, these tokens cannot be used with Cognito User Pools anymore. However, they are still valid when used with other services like AppSync or API Gateway.
+
+For limiting subsequent calls to these other services after invalidating tokens, we recommend lowering token expiration time for your app client in the Cognito User Pools console. If you are using the Amplify CLI this can be accessed by running `amplify console auth`.
+
+Token revocation is enabled automatically on new Amazon Cognito User Pools, however existing User Pools must enable this feature, [using the Cognito Console or AWS CLI](https://docs.aws.amazon.com/cognito/latest/developerguide/token-revocation.html)
```
Comment (tjleing) [inline docs/lib/auth/fragments/native_common/signout/common.md:15]: Maybe
```suggestion
Token revocation is enabled automatically on new Amazon Cognito User Pools, however existing User Pools must enable this feature, [using the Cognito Console or AWS CLI](https://docs.aws.amazon.com/cognito/latest/developerguide/token-revocation.html).
```

**Review by tjleing — APPROVED** [review]
Ran the migration script so that these changes will show up on live without needing another PR.  Looks good @div5yesh!

**Comment by div5yesh** [comment]
Migrated to new docs: https://github.com/aws-amplify/docs/pull/4230

**Pull request closed without merging**